lib/python/rose/apps/rose_ana.py

```diff
@@ -80,7 +85,11 @@ def enter_comparison(
         # the "comparisons" table
         sql_statement = (
             "INSERT OR REPLACE INTO comparisons VALUES (?, ?, ?, ?, ?)")
-        sql_args = [app_task, kgo_file, suite_file, status, comparison]
+        # Prepend the task_name onto each entry, to try and ensure it is
+        # unique (the individual comparison names may not be, but the rose
+        # task name + the comparison task name should)
+        sql_args = [self.task_name + " - " + app_task,
```

Comment:
I can see that it looks that way, but despite the argument names both being `app_task`, one is set by `rose_ana` itself to the Rose task name (the one which is saved to `self.task_name`) but the other one (passed to `enter_comparison`) is supplied by the user, and can be anything - though by convention it uses the section name from the app...  I could perhaps rename the arguments to make this more obvious?